Source/EasyNetQ/RabbitBus.cs

```diff
@@ -163,7 +162,7 @@ public class RabbitBus : IBus
                 maxPriority: configuration.MaxPriority,
                 maxLength: configuration.MaxLength,
                 maxLengthBytes: configuration.MaxLengthBytes);
-            var exchange = advancedBus.ExchangeDeclare(exchangeName, ExchangeType.Topic);
+            var exchange = publishExchangeDeclareStrategy.DeclareExchange(typeof(T), ExchangeType.Topic);
```

Comment:
It is a breaking change for existing code which use custom conventions.

Reply:
Conventions are used inside publishExchangeDeclareStrategy. So naming conventions does not change. And current behavior (different conventions for publish and subscribe) looks more like a bug, not an intentional one.

Reply:
this makes total sense to me